dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/inbound/HttpInputStreamImpl.java

```diff
@@ -106,7 +106,7 @@ public HttpInputStreamImpl(HttpInboundServiceContext context, FullHttpRequest re
     }
 
     private boolean isCompressed(String encoding) {
-        return "gzip".equalsIgnoreCase(encoding) || "deflate".equalsIgnoreCase(encoding);
+        return "gzip".equalsIgnoreCase(encoding) || "deflate".equalsIgnoreCase(encoding) || "x-gzip".equalsIgnoreCase(encoding);
```

Comment:
Where is this check done in Legacy? Just making sure we're not missing anything else.

Reply:
We could turn it into an enum lookup, but this [constants class](https://github.com/OpenLiberty/open-liberty/blob/c429b53d0b07db3929cd7947230429b8a7a37138/dev/com.ibm.ws.transport.http/src/com/ibm/wsspi/http/channel/HttpConstants.java#L42) already has them.